drivers/hud_apr/app/models/hud_apr/generators/shared/fy2023/question_five.rb

```diff
@@ -0,0 +1,179 @@
+          clause: leavers_clause,
+        },
+        # Number of adult leavers
+        {
+          row: '7',
+          clause: leavers_clause.and(adult_clause),
+        },
+        # Number of adult and HoH leavers
+        {
+          row: '8',
+          clause: leavers_clause.
+            and(adult_clause.
+              or(a_t[:head_of_household].eq(true))),
+        },
+        # Number of stayers
+        {
+          row: '9',
+          clause: stayers_clause,
+        },
+        # Number of adult stayers
+        # must match Q16-C14
+        {
+          row: '10',
+          clause: stayers_clause.
+            and(adult_clause),
+        },
+        # Number of veterans
+        {
+          row: '11',
+          clause: veteran_clause,
+        },
+        # Number of chronically homeless
+        {
+          row: '12',
+          clause: a_t[:chronically_homeless].eq(true),
+        },
+        # Number of youth under 25
+        {
+          row: '13',
+          clause: a_t[:age].lt(25).
+            and(a_t[:age].gteq(12)).
+            and(a_t[:other_clients_over_25].eq(false)),
+        },
+        # Number of parenting youth under 25 with children
+        {
+          row: '14',
+          clause: a_t[:age].lt(25).
+            and(a_t[:age].gteq(12)).
+            and(a_t[:parenting_youth].eq(true)),
+        },
+        # Number of adult HoH
+        {
+          row: '15',
+          clause: adult_clause.
+            and(a_t[:head_of_household].eq(true)),
+        },
+        # Number of child and unknown age HoH
+        {
+          row: '16',
+          clause: a_t[:age].lt(18).
+            or(a_t[:age].eq(nil)).
+            and(a_t[:head_of_household].eq(true)),
+        },
+      ]
+    end
+
+    private def q5_validations
+      table_name = 'Q5a'
+      metadata = {
+        header_row: TABLE_HEADER,
+        row_labels: ROW_LABELS,
+        first_column: 'B',
+        last_column: 'C',
+        first_row: 2,
+        last_row: 17,
+      }
+      @report.answer(question: table_name).update(metadata: metadata)
+
+      ['B', 'C'].each do |col|
+        # Column B is limited to engaged clients for SO, column C is not
+        inclusion_clause = if col == 'B'
+          engaged_clause
+        else
+          Arel.sql('1=1')
+        end
+        # Total clients
+        cell = "#{col}2"
+        answer = @report.answer(question: table_name, cell: cell)
+
+        members = universe.members.where(inclusion_clause)
+        answer.add_members(members)
+        answer.update(summary: members.count)
+
+        active_questions.each do |data|
+          cell = "#{col}#{data[:row]}"
+          answer = @report.answer(question: table_name, cell: cell)
+          members = universe.members.where(inclusion_clause).where(data[:clause])
+          answer.add_members(members)
+          answer.update(summary: members.count)
+        end
+
+        # HoH and adult stayers in project 365 days or more
```

Comment:
I'm confused why this is pulled out from `active_questions`?

Reply:
will do!